### DISCUSSION TITLE
How to enable usage of screen keyboard?

### DISCUSSION THREAD

**Discussion by Intevel** [microsoft/WSA#491] [General]
Android has an on-screen keyboard just like other operating systems, how do I activate/open it in a WSA application? If no keyboard is connected to my computer.

**Comment by zoey29o** [comment]
> Android has an on-screen keyboard just like other operating systems, how do I activate/open it in a WSA application? If no keyboard is connected to my computer.

well for that you need to install another launcher of your choice.  the downside you won't have notifications if ou done have the main window open. if you use on screen keyboard in windows it won't make a diffence since for android it would be the same
